index.html

```diff
@@ -11,9 +11,52 @@
         integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
     <link href="./css/index.css" rel="stylesheet" type="text/css">
+    
 </head>
 
 <body>
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+        <link rel="stylesheet" href="css/reg.css">
+    
+    </head>
+    <body>
+        
+        <div class="box">
+            
+            <div class="header-text">
+                <h1>Get In Touch</h1>
+            </div>
+        <hr>
+            <div class="form-fields">
+                <label for="fname">Enter Name</label>
+                <input type="text" id="fname" placeholder="Enter Name Here" >
+    
+    
+                <label for="email">Enter Email</label>
+                <input type="email" id="email" placeholder="Enter Email Here" >
+    
+                <label for="mob" class="test">Mobile No</label>
+                <input type="number" id="mob" placeholder="Enter mobile No" >
+    
+                <label for="msg">Message Title</label>
+                <input type="text" id="msg" placeholder="Message Title" >
+
+                <label for="Text">Write Message</label>
+                <input type="text1" id="Text" placeholder="Enter Here" >
+
+               <input type="submit" value="Submit">
```

Comment:
Add validations and submit action.

Reply:
Validations added successfully using JavaScript
@shubham-8551 now u can review this task.